crates/forge/tests/e2e/running.rs

```diff
@@ -806,6 +808,24 @@ fn validate_init(temp: &TempDir, validate_snforge_std: bool) {
     assert_stdout_contains(output, expected);
 }
 
+fn validate_gitignore(temp: &TempDir, path: &str) {
```

Comment:
you don't need to pass the path as separate parameter, you can just pass the path as one

Reply:
done